pkg/api/serialization_test.go

```diff
@@ -51,7 +51,7 @@ func originFuzzer(t *testing.T, seed int64) *fuzz.Fuzzer {
 		// Roles and RoleBindings maps are never nil
 		func(j *authorizationapi.Policy, c fuzz.Continue) {
 			c.FuzzNoCustom(j)
-			if j.Roles != nil {
+			if j.Roles == nil {
```

Comment:
If the comment "Roles and RoleBindings maps are never nil" is true, do we need this checks at all?

Reply:
The comment is from the perspective of the API.  Hence the nil check is there to make the fuzzer agree with the API.